crate/pkcs11/provider/src/kms_object.rs

```diff
@@ -137,31 +144,182 @@ async fn locate_objects(
         ..Default::default()
     };
     let response = kms_rest_client.locate(locate).await?;
+    debug!("Locate response: ids: {:?}", response.unique_identifiers);
     let uniques_identifiers = response
         .unique_identifiers
         .unwrap_or_default()
         .iter()
         .map(std::string::ToString::to_string)
         .filter(|id| !id.is_empty())
         .collect();
+    debug!("Located objects: tags: {tags:?} => {uniques_identifiers:?}");
+    Ok(uniques_identifiers)
+}
+
+pub(crate) fn kms_create(
+    kms_rest_client: &KmsClient,
+    algorithm: KeyAlgorithm,
+    key_length: usize,
+    sensitive: bool,
+    label: Option<&str>,
+) -> Pkcs11Result<KmsObject> {
+    tokio::runtime::Runtime::new()?.block_on(kms_create_async(
+        kms_rest_client,
+        algorithm,
+        key_length,
+        sensitive,
+        label,
+    ))
+}
+
+pub(crate) async fn kms_create_async(
```

Comment:
this only creates a symkey no ?
maybe rename this to be more precise

Reply:
yes, renamed to `kms_import_symmetric_key`